index.js

```diff
@@ -51,33 +54,37 @@ module.exports = function (log, indexesPath) {
   const queriesActive = Obv().set(0)
 
   loadIndexes(() => {
-    debug('loaded indexes', Object.keys(indexes))
+    debug('loaded indexes', [...indexes.keys()])
 
-    if (!indexes['seq']) {
-      indexes['seq'] = {
+    if (!indexes.has('seq')) {
+      indexes.set('seq', {
         offset: -1,
         count: 0,
         tarr: new Uint32Array(16 * 1000),
         version: 1,
-      }
+      })
     }
-    if (!indexes['timestamp']) {
-      indexes['timestamp'] = {
+    if (!indexes.has('timestamp')) {
+      indexes.set('timestamp', {
         offset: -1,
         count: 0,
         tarr: new Float64Array(16 * 1000),
         version: 1,
-      }
+      })
     }
-    if (!indexes['sequence']) {
-      indexes['sequence'] = {
+    if (!indexes.has('sequence')) {
+      indexes.set('sequence', {
         offset: -1,
         count: 0,
         tarr: new Uint32Array(16 * 1000),
         version: 1,
-      }
+      })
     }
 
+    seqIndex = indexes.get('seq')
+    timestampIndex = indexes.get('timestamp')
+    sequenceIndex = indexes.get('sequence')
```

Comment:
Using `seqIndex` directly instead of `indexes['seq']` or `indexes.get('seq')` could be a performance improvement. Not a big one, but given that we have `indexes['seq']` on hot code path, this could make a difference.